packages/analytics/src/helpers.ts

```diff
@@ -304,8 +304,12 @@ function wrapGtag(
           gtagParams as GtagConfigOrEventParams
         );
       } else if (command === GtagCommand.CONSENT) {
-        const [gtagParams] = args;
-        gtagCore(GtagCommand.CONSENT, 'update', gtagParams as ConsentSettings);
+        const [consentString, gtagParams] = args;
+        gtagCore(
+          GtagCommand.CONSENT,
+          consentString,
```

Comment:
how about consentAction or consentMethod? The allowed values are "update" and "default" so maybe also a comment mentioning the allowed values.

Reply:
Done!